code/modules/mob/living/simple_animal/abnormality/he/red_queen.dm

```diff
@@ -0,0 +1,49 @@
+/mob/living/simple_animal/hostile/abnormality/red_queen
+	name = "Red Queen"
+	desc = "A noble red abnormality sitting in her chair."
+	icon = 'ModularTegustation/Teguicons/48x64.dmi'
+	icon_state = "redqueen"
+	maxHealth = 650
+	health = 650
+	threat_level = HE_LEVEL
+	work_chances = list(
+
+		//All at 90, this is supposed to be guaranteed, due to her secondary effect
+		ABNORMALITY_WORK_INSTINCT = list(90),
+		ABNORMALITY_WORK_INSIGHT = list(90),
+		ABNORMALITY_WORK_ATTACHMENT = list(90),
+		ABNORMALITY_WORK_REPRESSION = list(90)
+		)
+	work_damage_amount = 15
+	work_damage_type = PALE_DAMAGE		//She doesn't really hurt you but I guess if she ever does she'll fuck you
+
+	ego_list = list(
+		)
+	var/liked
+
+
+/mob/living/simple_animal/hostile/abnormality/red_queen/Initialize(mapload)
+	liked = pick(ABNORMALITY_WORK_INSTINCT, ABNORMALITY_WORK_INSIGHT, ABNORMALITY_WORK_ATTACHMENT, ABNORMALITY_WORK_REPRESSION)
+	//What does she like?
+	//Pick it once so people can find out
+
+
+/mob/living/simple_animal/hostile/abnormality/red_queen/work_complete(mob/living/carbon/human/user, work_type, pe)
+	..()
+	if(work_type != liked)
+		if(prob(20))
+
+			//The Red Queen is fickle, if you're unlucky, fuck you.
+			user.apply_damage(200, RED_DAMAGE, null, user.run_armor_check(null, RED_DAMAGE), spread_damage = TRUE)
```

Comment:
Add a sound + effect and message here, so people around them can see what happened even if they don't get beheaded.